pkg/promutil/test/multiprojects_metric_scences.go

```diff
@@ -0,0 +1,538 @@
+package main
+
+import (
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/hanfei1991/microcosm/pkg/promutil"
+	"github.com/hanfei1991/microcosm/pkg/tenant"
+	"github.com/pingcap/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func main() {
```

Comment:
Do we need to run this test in CI?

Reply:
Seems unnecessary.
I use this test to observe whether promethus and grafana can work normally.
I don't know how to check the promethus and grafana if we run this test in CI and what we expect from this ci.